src/components/typings/Button.ts

```diff
@@ -1,6 +1,12 @@
 import * as React from "react";
 
-export type ButtonType = "primary" | "secondary" | "link" | "success" | "alert";
+export type ButtonType =
+  | "primary"
+  | "secondary"
+  | "link"
+  | "success"
+  | "alert"
+  | "dropdown";
```

Comment:
Don't add this in default button. The problem is that we are not providing full functionality of dropdown there